g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/validation/ParameterValidation.java

```diff
@@ -17,6 +21,13 @@
 
 public class ParameterValidation implements IVlidation {
 	HttpServletRequest request = null;
+	public final static String TAG_BUNDLE_BASE_PATH = "bundleBasedPath";
+	public final static String BUNDLE_FILE = "bundle.json";
+
+	/**
+	 * A cached map to store the data of product white list
+	 */
+	private  static final Map<String, Object> MAP_PRODUCTS_VERSIONS = new HashMap<>();
```

Comment:
Can this white list be updated at runtime?

Reply:
it requires to reboot vip service to reload the cache.